docs-ref-services/latest/resourcemanager-iothub-readme.md

```diff
@@ -1,14 +1,13 @@
 ---
 title: Azure Resource Manager IotHub client library for Java
+description: This package contains Microsoft Azure SDK for IotHub Management SDK. Use this API to manage the IoT hubs in your Azure subscription.
 keywords: Azure, java, SDK, API, azure-resourcemanager-iothub, iothub
-author: ramya-rao-a
-ms.author: ramyar
+author: sima-zhu
+ms.author: sizhu
```

Comment:
is this a required field?  I wish we didn't leak people's email addresses in these documents